src/Hazelcast.Net/Clustering/DistributedEventScheduler.cs

```diff
@@ -49,12 +56,14 @@ public DistributedEventScheduler(ILoggerFactory loggerFactory)
             // TODO: how many queues should we retain?
             const int size = 10;
             _pool = new SimpleObjectPool<Queue>(() => new Queue(), size);
+
+            HConsole.Configure(options => options.Set(this, x => x.SetPrefix("SCHEDULER")));
         }
 
         /// <summary>
         /// Triggers when an event handler throws an exception.
         /// </summary>
-        public event EventHandler<DistributedEventExceptionEventArgs> OnError;
+        public event EventHandler<DistributedEventExceptionEventArgs> HandlerError;
```

Comment:
Maybe?
```suggestion
        public event EventHandler<DistributedEventExceptionEventArgs> HandleError;
```

Reply:
Ah no, it's an event that triggers when a _handler_ produces an _error_, thus it's a _HandlerError_ event.